api/rosetta/data_balance_integration_test.go

```diff
@@ -90,69 +104,79 @@ func TestGetBalance(t *testing.T) {
 	db := setupDB(t)
 	api := setupAPI(t, db)
 
+	const (
+		testAccount = "754aed9de6197641"
+	)
+
+	var (
+		// block where the account first appears
+		first = identifier.Block{
+			Index: 13,
+			Hash:  "af528bb047d6cd1400a326bb127d689607a096f5ccd81d8903dfebbac26afb23",
+		}
+
+		// a block mid-chain
+		second = identifier.Block{
+			Index: 50,
+			Hash:  "d99888d47dc326fed91087796865316ac71863616f38fa0f735bf1dfab1dc1df",
+		}
+
+		// last indexed block
+		last = identifier.Block{
+			Index: 425,
+			Hash:  "594d59b2e61bb18b149ffaac2b27b0efe1854f6795cd3bb96a443c3676d78683",
+		}
```

Comment:
In this case, I would actually use `firstBlock`, `secondBlock`, `lastBlock`, because it's not clear anymore what it is when you use it in `balanceRequest(testAccount, first)`. When you use `balanceRequest(testAccount, firstBlock)`, it's instantly obvious.